xrootd/xrdproto/dirlist/dirlist.go

```diff
@@ -20,18 +20,63 @@ import (
 const RequestID uint16 = 3004
 
 // Response is a response for the dirlist request,
-// which contains a slice of entries containing the entry name and the entry stat info.
+// which contains a slice of entries containing the entry name and the entry stat info,
+// and a WithStatInfo flag which indicates whether a request asked for a stat info.
```

Comment:
`s/which indicates whether/indicating whether/`
(to mirror the wording of the sentence above, for the `Entries` doc.)